package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "__BONES_NEEDS_A_NAME__",
+  "name": "dogbones",
```

Comment:
You know what, on second thought, `dogbones` is too cute. Let's just do `spooky`. Because that's not cute at all.